File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.py
class Solution(object):
    def getMinDistance(self, nums, target, start):
        midset=9999999999
        n=len(nums)
        for i in range(0,n):
            if(nums[i]==target):
                midset=min(midset,abs(i-start))
        return midset        
        

File: 1975-minimum-distance-to-the-target-element/1975-minimum-distance-to-the-target-element.java
class Solution {
    public int getMinDistance(int[] nums, int target, int start) {
        int midest=Integer.MAX_VALUE;
        int n=nums.length;
        for(int i=0;i<n;i++)
        { 
            if(nums[i]==target)
            {
                midest=Math.min(midest,Math.abs(i-start));
            }

        }
        return midest;
        
    }
}